orbit/pkg/table/user_exec/exec.go

```diff
@@ -0,0 +1,144 @@
+// build +darwin
+
+package user_exec
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"os/user"
+	"time"
+
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/osquery/osquery-go/plugin/table"
+	"github.com/rs/zerolog/log"
+)
+
+// ExecOsqueryLaunchctl runs osquery under launchctl, in a user context.
+func ExecOsqueryLaunchctl(ctx context.Context, timeoutSeconds int, username string, osqueryPath string, query string) ([]byte, error) {
+	ctx, cancel := context.WithTimeout(ctx, time.Duration(timeoutSeconds)*time.Second)
+	defer cancel()
+
+	targetUser, err := user.Lookup(username)
+	if err != nil {
+		return nil, fmt.Errorf("looking up username %s: %w", username, err)
+	}
+
+	cmd := exec.CommandContext(ctx,
+		"launchctl",
```

Comment:
Do we need/want `launchctl` packaged in Orbit to be able to run osqueryd as user?

Reply:
launchctl is going to be specific to the version of launchd and thus macos version. It is also
always going to be on macOS so I don't think it's necessary.